src/Assigner/ProductsAssignerInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * another great project.
+ * You can find more information about us on https://bitbag.shop and write us
+ * an email on mikolaj.krol@bitbag.pl.
+ */
+
+declare(strict_types=1);
+
+namespace Loevgaard\SyliusBrandPlugin\Assigner;
+
+use Loevgaard\SyliusBrandPlugin\Entity\BrandAwareInterface;
+use Loevgaard\SyliusBrandPlugin\Entity\ProductsAwareInterface;
+
+interface ProductsAssignerInterface
+{
+    /**
+     * @param ProductsAwareInterface $productsAware
+     * @param BrandAwareInterface[] $products
```

Comment:
What is the reason this needs to be `BrandAwareInterface[]` and not just `ProductInterface[]`?

Reply:
As I say on a previous comment you should keep in mind that those class will be overridden to feet the project need. If someone want to override it you have to keep the smallest interface signature you have.